### PR TITLE
release-schema.json: convert unofficialTranslation from boolean to ar…

### LINKED ISSUE #189: BT-708 and BT-737 official and nonofficial languages in tender documents
BT-737-Lot/Part, `efac:NonOfficialLanguages/cac:Language/cbc:ID` is repeatable within `cac:CallForTendersDocumentReference` as is BT-708-Lot/Part, `efac:OfficialLanguages/cac:Language/cbc:ID`. So a single document can have multiple languages noted as it's official languages and as it's unofficial languages.

e.g. (taken from [pin-only_24-maximal.xml](https://github.com/OP-TED/eForms-SDK/blob/develop/examples/notices/pin-only_24_maximal.xml))

```xml
<cac:CallForTendersDocumentReference>
  <ext:UBLExtensions>
    <ext:UBLExtension>
      <ext:ExtensionContent>
        <efext:EformsExtension>
          <efac:NonOfficialLanguages>
            <cac:Language>
              <cbc:ID>CAT</cbc:ID>
            </cac:Language>
          </efac:NonOfficialLanguages>
          <efac:OfficialLanguages>
            <cac:Language>
              <cbc:ID>ENG</cbc:ID>
            </cac:Language>
            <cac:Language>
              <cbc:ID>FRA</cbc:ID>
            </cac:Language>
          </efac:OfficialLanguages>
        </efext:EformsExtension>
      </ext:ExtensionContent>
    </ext:UBLExtension>
  </ext:UBLExtensions>
  <cbc:ID>Doc1</cbc:ID>
  <cbc:DocumentType>non-restricted-document</cbc:DocumentType>
</cac:CallForTendersDocumentReference>
```

Currently in guidance.yaml we have:
> **BT-708**
>  [Get the document for the document reference](operations.md#get-the-document-for-a-document-reference).
> Lowercase and convert the code into a two-letter [ISO 639-1 code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes), and add it to the document's `.languages` array.

Which allows for multiple languages but doesn't specify that they are official.

>**BT-737**
> [Get the document for the document reference](operations.md#get-the-document-for-a-document-reference).
> Set the document's `.unofficialTranslation` to `true`.

Which doesn't actually map the language code anywhere.
Following this guidance for the above snippet of XML we'd get the following OCDS:

```json
{
  "documents": [
    {
      "id": "Doc1",
      "languages": [
	 "en",
         "fr"
      ],
      "unofficialTranslation": true
    }
  ]
}
```

which both misses out the unofficial translation language and gives the erroneous impression that the languages given are unofficial.

### PR DESCRIPTION
**Pull request by odscjen**
…ray of languages

closes https://github.com/open-contracting/european-union-support/issues/189

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 * Indicate the page numbers at which relevant information can be found within a large document
 * Describe any special arrangements needed to access the document
 * Name the author of the document (not to be confused with its publisher)
-* Indicate whether the document is an unofficial translation
+* Provide any unofficial languages the document is provided in
```
Comment (duncandewhurst) [inline README.md]:
```suggestion
* Indicate the languages in which unofficial translations of the document are available
```

File: release-schema.json [hunk 2]
```diff
@@ -47,13 +47,21 @@
           ],
           "minLength": 1
         },
-        "unofficialTranslation": {
-          "title": "Unofficial translation",
-          "description": "Whether the document is an unofficial translation.",
+        "unofficialTranslations": {
+          "title": "Unofficial translations",
+          "description": "The languages used in unofficial translations of the document, from the open [language](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#language) codelist. A [BCP47 language tag](https://www.w3.org/International/articles/language-tags/) is allowed, if there is a user need for the additional information.",
```
Comment (duncandewhurst) [inline release-schema.json]:
```suggestion
          "description": "The languages in which unofficial translations of the document are available, from the open [language](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#language) codelist. A [BCP47 language tag](https://www.w3.org/International/articles/language-tags/) is allowed, if there is a user need for the additional information.",
```

**Review by duncandewhurst — CHANGES_REQUESTED** [review]
If you're happy with my suggestions, you can request a review from James once you've committed them.

**Review by jpmckinney — APPROVED** [review]

**Comment by jpmckinney** [comment]
Ah, we need to add the `language.csv` codelist to this extension, which is only introduced in 1.2.

**Pull request merged**